Maestro/subscriptions.json

```diff
@@ -6,6 +6,13 @@
       "vsoProject": "DevDiv",
       "buildDefinitionId": 1084
     },
+    // A build definition capable of running any .ps1 script in the CoreFX repo dev/api branch
+    "corefx-general-dev-api": {
+      "vsoInstance": "devdiv.visualstudio.com",
+      "vsoProject": "DevDiv",
+      "vsoSourceBranch": "dev/api",
```

Comment:
Ah, I actually think putting it in the subscription is better. Adding a new action was just my first thought because I was expecting to have to clone the definition, and when I saw the `vsoSourceBranch` commit I changed it to this. Fixing.